tools/site-config/src/index.js

```
@@ -64,16 +64,24 @@ function getTargetOrg() {
 function getSiteConfig(packageName) {
     const target = getTargetOrg();
 
+    const sitePrefixes = {
+        '@api-extractor': 'https://api-extractor.com',
+        '@rushjs': 'https://rushjs.io',
+        '@tsdoc': 'https://tsdoc.org'
```

This design seems slightly error-prone.

What happens if the prefix is misspelled?  For example `@rush` instead of `@rushjs`?  Will `@rush` silently get included in the URL?

Or what if the token has additional characters, for example `@tsdocjs` instead of `@tsdoc`?  Would the matcher replace the substring and incorrectly preserve the unmatched `js` text?

1. Maybe we could adopt a convention like `@subsite(rush)`, `@subsite(ae)` and then the plugin could report an error if the `(` `)` token is not defined?
2. Or if we want to keep them short, maybe the matcher could look for a generalized RegExp like `/^@[a-z][a-z0-9_]*\b/i`, and then report an error if the tag name is not one of the defined strings (which can also accept the Docusaurus system tag names, assuming there's a known list of them).  We could also report an error if the immediately following character is not a `/`.

---

If you misspell a prefix, it will immediately fail the build (it throws an error inside the plugin).

---

(I do see what you mean about the possibility of a truncated prefix though, your suggestion of requiring "/" after the prefix seems like a simple fix there.)